docs/hooks.md

```diff
@@ -5,19 +5,46 @@ Heptio Ark currently supports executing commands in containers in pods during a
 ## Backup Hooks
 
 When performing a backup, you can specify one or more commands to execute in a container in a pod
-when that pod is being backed up. There are two ways to specify hooks: annotations on the pod
-itself, and in the Backup spec.
+when that pod is being backed up.
+
+Ark versions prior to v0.7.0 only support hooks that execute prior to any custom action processing
```

Comment:
I think it might be helpful to add an end-to-end outline of the steps in the backup process: first run pre hooks, then run custom actions one by one, backing up any additional items returned before moving on to the next custom action, then run post hooks, then add the item to the tarball. This would help users see the overall process and exactly where hooks fit in, which might be helpful if they're not intimately familiar with how the process proceeds.

I wonder if longer-term we should try to write out full-length docs describing in some detail how the backup and restore operations proceed. That might be helpful, but not necessary for now. cc @Bradamant3

Reply:
SGTM, but let's do it in another PR.

Reply:
OK.